packages/valory/skills/proposal_voter_abci/rounds.py

```diff
@@ -184,6 +184,9 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
                     synchronized_data_class=SynchronizedData,
                     **{
                         get_name(SynchronizedData.proposals): payload["proposals"],
+                        get_name(
+                            SynchronizedData.snapshot_proposals
+                        ): [],  # clear snapshot proposals to avoid too big registration payloads
```

Comment:
Drop snapshot proposals at the end of the period. We are retrieving them from scratch anyway on the next period, and this way we avoid too big registration payloads.